aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -1521,6 +1526,37 @@ open class AztecText : AppCompatEditText, TextWatcher, UnknownHtmlSpan.OnUnknown
         }
     }
 
+    fun addRefreshListenersToTaskLists(editable: Editable, start: Int, end: Int) {
+        val taskLists = editable.getSpans(start, end, AztecTaskListSpan::class.java)
+        taskLists.forEach { taskList ->
+            if (taskList.onRefresh == null) {
+                taskList.onRefresh = {
+                    refreshTaskListSpan(it)
+                }
+            }
+        }
+    }
+
+    private fun refreshTaskListSpan(taskList: AztecTaskListSpan) {
```

Comment:
This function removes and recreates the task list span. This is the only way to redraw anything in a Spannable. we cannot have animations for the redrawing.

Reply:
yep, we only refresh it on user click on the checkbox or around it